src/rust-2021/prelude.md

```diff
@@ -31,5 +30,85 @@ It's identical to the current one, except for three new additions:
 - [`std::convert::TryFrom`](https://doc.rust-lang.org/stable/std/convert/trait.TryFrom.html)
 - [`std::iter::FromIterator`](https://doc.rust-lang.org/stable/std/iter/trait.FromIterator.html)
 
-The library team still needs to formally approve these, which will likely happen soon.
-<!-- TODO: hopefully this happens before we publish this -->
+The tracking issue [can be found here](https://github.com/rust-lang/rust/issues/85684).
+
+## Migration to Rust 2021
+
+As a part of the 2021 edition a migration lint, `rust_2021_prelude_collisions`, has been added in order to aid in automatic migration of Rust 2018 codebases to Rust 2021.
+
+In order to have rustfix migrate your code to be Rust 2021 Edition compatible, run:
+
+```ignore
```

Comment:
Generally it is preferred to use a specific language like console, sh, or plaintext.

```suggestion
```sh
```